__tests__/app.test.js

```diff
@@ -112,4 +114,78 @@ describe('API', () => {
                 });
         });
     });
+
+    describe('/API/ARTICLES/:ARTICLE_ID/COMMENTS', () => {
+        test('Status 200 - api point exists and responds', () => {
+            return request(app).get("/api/articles/1/comments").expect(200);
+        });
+        test('Status 200 - returns back an object and has a property called comments', () => {
+            return request(app)
+                .get("/api/articles/1/comments")
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body).toBeInstanceOf(Object);
+                    expect(body).toHaveProperty("comments");
+                    expect(body.comments).toBeInstanceOf(Array);
+                });
+        });
+        test('Status 200 - returns back an array of objects with the correct keys', () => {
+            return request(app)
+                .get("/api/articles/1/comments")
+                .expect(200)
+                .then(({ body }) => {
+                    const { comments } = body;
+
+                    expect(comments).toHaveLength(11);
+                    
+                    comments.forEach( comment => {
+                        expect(comment).toEqual(
+                            expect.objectContaining({
+                                comment_id: comment.comment_id,
+                                body: comment.body,
+                                article_id: comment.article_id,
+                                author: comment.author,
+                                votes: comment.votes,
+                                created_at: comment.created_at,
+                            })
+                        )
+                    });
+                });
+        });
+        test('Status 200 - returns back an array of comments from DB', () => {
+            const expectedTopComment = {
+                comment_id: 2,
+                body: 'The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.',
+                article_id: 1,
+                author: 'butter_bridge',
+                votes: 14,
+                created_at: "2020-10-31T03:03:00.000Z"
+              };
+
+            return request(app)
+                .get("/api/articles/1/comments")
+                .expect(200)
+                .then(({ body }) => {
+                    const topBody = body.comments[0];
+
```

Comment:
We also want another Status 200 Test for when the ID is valid, but the comments respond with an empty array